src/components/views/rooms/MessageComposer.tsx

```diff
@@ -195,6 +196,8 @@ class UploadButton extends React.Component<IUploadButtonProps> {
 
 interface IProps {
     room: Room;
+    // TODO: This should be provided via some sensible global context instead.
```

Comment:
What do you have in mind here? I'm not sure I would understand how to action this TODO?

Reply:
That's a leftover from previous experiments and it should be taken from the RoomContext instead. Thanks for catching!